scripts/find_pybind11

```diff
@@ -0,0 +1,44 @@
+#!/usr/bin/env python
+
+import os,sys
+
+pybind11_g = "pybind11"
+
+def _find_src(starting_dir):
+
```

Comment:
Tiny winy details, can you add the doc strings?

Reply:
youp, this would be indeed a good habit for me to get. I'll run the format too

Reply:
Super! thank you very much!